draft-ietf-quic-tls.md

```diff
@@ -548,47 +548,50 @@ use.
 ### TLS Interface Summary
 
 {{exchange-summary}} summarizes the exchange between QUIC and TLS for both
-client and server. Each arrow is tagged with the encryption level used for that
-transmission.
+client and server. Solid arrows indicate packets that carry CRYPTO frames;
```

Comment:
Technically speaking, 1-RTT packets can carry CRYPTO frames, too.

Reply:
Yes, well technically this doesn't say that they don't, only that these do. :)

Reply:
Touché.

Reply:
Why not say "handshake data", since you say "application data" later?